### DISCUSSION TITLE
Showing off: New project for casting and modeling agencies

### DISCUSSION THREAD

**Discussion by blendsdk** [microsoft/fluentui#25546] [Show and tell]
Hi all,

I have been working on a new product which offers workflow for casting and modeling agencies. Here are some screenshots to show how customizable FluentUI React can be. This application has its own custom theme generated using [the FluentUI theme builder](https://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/heads/master/theming-designer/). The backend is built on top of [PostgreSQL](https://www.postgresql.org/) and BlendSDK (***our own custom NodeJS framework and code generator***)

Have fun coding.

## Dashboard
![](https://gist.githubusercontent.com/blendsdk/1dcad051db496ce2eefed3780ad9cf49/raw/1f792ad6cc8dfe5e79b18a765789e539ff1186fe/8.png)

## Module navigation
**Normally we use this to navigate between main modules of an application**
![](https://gist.githubusercontent.com/blendsdk/1dcad051db496ce2eefed3780ad9cf49/raw/1f792ad6cc8dfe5e79b18a765789e539ff1186fe/9.png)

## Custom Tile Component
**I have created a custom TileList component inspired from the examples available in this repository**
![](https://gist.githubusercontent.com/blendsdk/1dcad051db496ce2eefed3780ad9cf49/raw/1f792ad6cc8dfe5e79b18a765789e539ff1186fe/10.png)

## TileList hosting image Tiles
![](https://gist.githubusercontent.com/blendsdk/1dcad051db496ce2eefed3780ad9cf49/raw/e8f27714dcee3c7c8995c51d991fa6353ac8fe90/11.png)

## Sample of a Tile implementation
![](https://gist.githubusercontent.com/blendsdk/1dcad051db496ce2eefed3780ad9cf49/raw/e8f27714dcee3c7c8995c51d991fa6353ac8fe90/12.png)

**Comment by khmakoto** [comment]
This is awesome @blendsdk! Thanks for sharing!

**Comment by tombohub** [comment]
This is fluent v8 theme builder, right?

**Reply by blendsdk** [reply]
> This is fluent v8 theme builder, right?

Yes, I have used the v8 theme builder
